core/src/main/java/org/keycloak/crypto/AsymmetricSignatureVerifierContext.java

```diff
@@ -51,4 +60,25 @@ public boolean verify(byte[] data, byte[] signature) throws VerificationExceptio
         }
     }
 
+    public byte[] concatenatedRSToASN1DER(final byte[] signature, int signLength) throws IOException {
```

Comment:
This should be static method, and should be close to ECDSA code, e.g. ECDSASignatureProvider.